src/index.html

```diff
@@ -8,6 +8,76 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <span class="stars__star"></span>
+
+      <span class="stars__star"></span>
+
+      <span class="stars__star"></span>
+
+      <span class="stars__star"></span>
+
+      <span class="stars__star"></span>
```

Comment:
no needs to add an empty line between oneline siblings
```suggestion
      <span class="stars__star"></span>
      <span class="stars__star"></span>
      <span class="stars__star"></span>
      <span class="stars__star"></span>
      <span class="stars__star"></span>
```